ble/GattServer.h

```diff
@@ -302,6 +322,10 @@ class GattServer {
         return BLE_ERROR_NONE;
     }
 
+    DataReadCallbackChain_t& onDataRead() {
```

Comment:
please add a documentation header. it might be good to add a code-example showing its use